src/pages/workspaces/workspace/analysis/ContextBar.js

```diff
@@ -218,7 +218,16 @@ export const ContextBar = ({
           tooltipDelay: 100,
           useTooltipAsLabel: false,
           ...Utils.newTabLinkProps
-        }, [icon('terminal', { size: 40 }), span({ className: 'sr-only' }, ['Terminal button'])])
+        }, [icon('terminal', { size: 40 }), span({ className: 'sr-only' }, ['Terminal button'])]),
+        h(Clickable, {
```

Comment:
This is enabled for Azure workspaces, but results in an error.

Reply:
Also, should this be behind a feature flag?

Reply:
Oops, didn't consider Azure workspaces. And yes, I forgot to add the feature flag.

Reply:
Remember to disable on Azure as well as the feature flag… and let me know if you figure out how to make the keyboard navigation more natural. These roles likely would help: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/treeitem_role.

Reply:
Thanks for the reminder. I'm working on updating this to use tree roles and add keyboard nav.